plugins/modules/device_credential_workflow_manager.py

```diff
@@ -7,23 +7,25 @@
 from __future__ import absolute_import, division, print_function
 
 __metaclass__ = type
-__author__ = ['Muthu Rakesh, Madhan Sankaranarayanan']
+__author__ = ['Muthu Rakesh, Madhan Sankaranarayanan, Megha Kandari']
 
 DOCUMENTATION = r"""
 ---
 module: device_credential_workflow_manager
 short_description: Resource module for Global Device Credentials and Assigning Credentials to sites.
 description:
-- Manage operations on Global Device Credentials and Assigning Credentials to sites.
+- Manage operations on Global Device Credentials, Assigning Credentials to sites and Sync Credentials to site device.
 - API to create global device credentials.
 - API to update global device credentials.
 - API to delete global device credentials.
 - API to assign the device credential to the site.
+- API to sync the device credential to the site. Sync functionality will be applicable for DNAC version 2.3.7.6 onwards.
```

Comment:
is applicable for Catalyst Center version 2.3.7.6 and later.

Reply:
Modified this.